monitoring/api/v3/alerts-client/snippets.py

```diff
@@ -0,0 +1,282 @@
+# Copyright 2018 Google Inc.
```

Comment:
Nit: LLC, not Inc. go/releasing/preparing#include-license-file-and-source-code-headers (Google Inc. doesn't exist now that Alphabet is the parent company.)

Reply:
Done.